example/templates/person.ts

```diff
@@ -118,7 +118,8 @@ const uischema = {
 const data = {
   name: 'John Doe',
   vegetarian: false,
-  birthDate: '1985-06-02'
+  birthDate: '1985-06-02',
+  personalData: {}
```

Comment:
@edgarmueller shouldn't nested objects be created on the fly when needed?

Reply:
@eneufeld Depends on how you define 'needed': The object is already created once you touch any of its properties, but one might also argue that we should already create an empty object even in case it's not actually required: if that's what you want, I'd propose to move that discussion to a different issue.

Reply:
I opened an issue to discuss this #708